api/Navigator.json

```diff
@@ -3132,6 +3132,55 @@
           }
         }
       },
+      "storage": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Navigator/storage",
+          "spec_url": "https://storage.spec.whatwg.org/#dom-navigatorstorage-storage",
+          "support": {
+            "chrome": {
+              "version_added": "55"
```

Comment:
This doesn't match https://developer.mozilla.org/en-US/docs/Web/API/StorageManager#browser_compatibility, can you investigate?

Reply:
It appears the collector changes the version from 48 to 55 for the StorageManager.  I'll send a PR to correct the data for StorageManager!

Reply:
Thanks!